service/teacher.go

```diff
@@ -15,7 +15,7 @@ func (s *Service) AddTeacher(teacher *models.Teacher) (string, error) {
 
 // RetrieveTeacher gets teacher from database
 func (s *Service) RetrieveTeacher(id string) (*models.Teacher, error) {
-	stu, err := s.rt.Db.GetTeacher(id)
+	stu, err := s.db.GetTeacher(id)
```

Comment:
For now as it's simple... we can simply do `return s.db.GetTeacher(id)`. No  need to check errors as we are not really doing anything for now. Same for all other Service functions i guess.